ocpp/v201/datatypes.py

```diff
@@ -454,7 +454,7 @@ class IdTokenType:
     """
 
     id_token: str
-    type: enums.IdTokenType
+    type: enums.IdTokenEnumType
```

Comment:
Not sure what's the value add of renaming enums

Reply:
Compliance with the OCPP specs

Reply:
Cross Checked this. This makes me question the whether the rest of the enums and datatypes are compliant or not. Will take a look at this soon.

Reply:
I did a double check when I came across this one and I couldn't find any others. But another set of eyes is always appreciated!